README.md

```diff
@@ -141,8 +142,35 @@ Plot all messages so far, with new messages each hour.
 </details>
 
 [↥ back to top](#table-of-contents)
+### Friend Request
+
+Plot the friends you make every day(Red) ,friend request send every day(green) and friend request received every day(blue)
+
+![](images/FriendEachDay.png)
+
+<details>
+  <summary>Plot all the friend request received,send and total friend so far.</summary>
+  </details>
+  
+Plot the cumalative friend you may every month,friend request send and friend request received.
+![](images/MonthwiseFriend.png)
+
+<details>
+  <summary>Plot the no.of friend request received,friend request send and total friend each month</summary>
+ </details>
+ 
+ Compare the Friend request recieved received and friend request send.
+ ![](images/ComparingFriendRequestReceivedorSend.png)
+ 
+ <details>
+  <summary>We compare monthwise no.of friend request send vs friend request received</summary>
+  </details>
+ 
+## Observations
```

Comment:
These observations should be added in the observations section, no need to create a new section.